Recruit-Training/Advanced-Recruit-Training/src/BatteryStateDisplayService.cpp

```diff
@@ -27,7 +32,9 @@ void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& ba
 
     QTextStream(stdout) << "Voltage: " << batteryData.voltage
         << " Current: " << batteryData.current
-        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed() << endl;
+        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed()
+        << " Time Until" << (batteryStateOfChargeService_.isCharging()? " Charged: " : " Depleted: ")
+        << batteryStateOfChargeService_.timeWhenChargedOrDepleted().toString(STRING_TIME_FORMAT) << endl;
 
     // TODO: Print out time until the battery is fully charged or depleted.
```

Comment:
For style, you can delete the comments, especially TODO ones if they've been done.